### DISCUSSION TITLE
Ignore certain properties from Vue's change detection wrapper

### DISCUSSION THREAD

**Discussion by alshdavid** [vuejs/petite-vue#161] [General]
There are certain properties that I have which cannot be wrapped in Proxies resulting in the following error:

```
Cannot create proxy with a non-object as target or handler
```

Is it possible to tell Petite Vue to ignore certain properties?

```javascript
class State {
  foo
  bar

  constructor(
    foo,
    bar,
  ) {
    this.foo = foo
    this.bar = bar
  }
}

const state = new State()
createApp(state, { skipChangeDetectionOn: ['foo'] }).mount(document.body)
```

**Comment by sqllyw** [comment]
if the properties you want to ignore are objects, then you can use Object.freeze, here is a sample:
https://jsfiddle.net/s6yu9xvm/
